src/main/java/mik/pet/project/model/dto/NewBookRequestDto.java

```diff
@@ -9,10 +11,17 @@
 @AllArgsConstructor
 @NoArgsConstructor
 public class NewBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
     private String isbn;
+    @NotNull
+    @Min(0)
     private BigDecimal price;
+    @NotNull
```

Comment:
Description and cover image are nullable fields if I'm not mistaken. Validate only min and max size of the field.